client-misk/src/main/kotlin/app/cash/backfila/client/misk/InstanceProvider.kt

```diff
@@ -13,6 +13,7 @@ import misk.hibernate.Transacter
 import misk.hibernate.shards
 import misk.hibernate.transaction
 import javax.persistence.Table
+import misk.hibernate.annotation.Keyspace as KeyspaceAnnotation
```

Comment:
seems like it would be confusing to rename this when importing it

Reply:
Unfortunately there is a Keyspace annotation and class. I have to name one of them or use the full path. Perhaps the full path then?